src/backwards-compatibility.js

```diff
@@ -135,6 +135,56 @@ Object.assign(pc.input, {
     TouchEvent: pc.TouchEvent
 });
 
+pc.log = {
+    write: function (text) {
+        console.log(text);
```

Comment:
should these also print out deprecated warnings?

Reply:
I considered it but thought it was weird that a console logging API, logging warning too. But yeah, TBH, it probably should.

Reply:
OK, I've added deprecation warnings for `pc.log.*`.